pkg/services/sqlstore/team.go

```diff
@@ -67,25 +67,15 @@ func getTeamMemberCount(filteredUsers []string) string {
 }
 
 func getTeamSelectSQLBase(filteredUsers []string) string {
-	return `SELECT
-		team.id as id,
-		team.org_id,
-		team.name as name,
-		team.email as email, ` +
-		getTeamMemberCount(filteredUsers) +
-		` FROM team as team `
-}
-
-func getTeamSelectWithPermissionsSQLBase(filteredUsers []string) string {
 	return `SELECT
 		team.id AS id,
 		team.org_id,
 		team.name AS name,
 		team.email AS email,
 		team_member.permission, ` +
 		getTeamMemberCount(filteredUsers) +
-		` FROM team AS team
-		INNER JOIN team_member ON team.id = team_member.team_id AND team_member.user_id = ? `
+		` FROM team AS team 
+		LEFT JOIN team_member ON team.id = team_member.team_id`
```

Comment:
the user filter is still needed here otherwise we'll return a row for every user:
```suggestion
		LEFT JOIN team_member ON team.id = team_member.team_id AND team_member.user_id = ?`
```
The change to LEFT join just means that we'll still get rows back when the user isn't a member

Reply:
Thanks!